subsys/bluetooth/host/iso.c

```diff
@@ -798,6 +798,57 @@ static bool valid_chan_io_qos(const struct bt_iso_chan_io_qos *io_qos,
 
 	return true;
 }
+
+int bt_iso_chan_get_tx_sync(const struct bt_iso_chan *chan, struct bt_iso_transmit_info *info)
+{
+	CHECKIF(chan == NULL) {
+		BT_DBG("chan is NULL");
+		return -EINVAL;
+	}
+
+	CHECKIF(chan->iso == NULL) {
+		BT_DBG("chan->iso is NULL");
+		return -EINVAL;
+	}
+
+	CHECKIF(info == NULL) {
+		BT_DBG("info is NULL");
+		return -EINVAL;
+	}
+
+	struct bt_hci_cp_le_read_iso_tx_sync *cp;
+	struct bt_hci_rp_le_read_iso_tx_sync *rp;
+	struct net_buf *buf;
+	struct net_buf *rsp = NULL;
+	int err;
```

Comment:
We typically declare variable at top of scope (i.e. before the `CHECKIF`s) in the host